src/services/item/repository.ts

```diff
@@ -141,7 +141,7 @@ export class ItemRepository extends MutableRepository<Item, UpdateItemBody> {
         ...settings,
       },
       // set lang from user lang
-      lang: lang ?? creator?.lang ?? 'en',
+      lang: lang ?? parent?.lang ?? creator?.lang ?? 'en',
```

Comment:
Should `en` be defined as the default language ? is it used in multiple places ? Maybe having it as a local constant in the file could give it more visibility ? Maybe it is fine to keep it like this.